src/components/sidebar/helpers/add-nav-item-meta-data.ts

```diff
@@ -12,6 +12,27 @@ interface AddNavItemMetaDataResult {
 	itemsWithMetadata: NavItemWithMetaData[]
 }
 
+/**
+ * Returns an object for rendering a Badge in a sidebar item if a <sup> tag is
+ * founnd in its title.
+ */
+const getBadgeFromTitle = (title: string) => {
```

Comment:
Is the plan to eventually update this in the source content? If so, maybe we leave some note breadcrumbs about this being temporary 🤔

Reply:
I don't think we've discussed a solid plan yet, but I'd love for this to be temporary! I think there's some work that'll need to be done first:

- Sunset the docs pages on .io sites
- Get content conformance checks in place, so we can maintain the rule of "no `<sup>` tags in nav data titles"
- Update all content to conform to the check (and IMO past versions of docs too, not just the latest)

Regardless, I'll add a comment so we know why it's here 😊